pkg/services/ngalert/models/alert_rule.go

```diff
@@ -113,6 +113,13 @@ type AlertRule struct {
 	Labels      map[string]string
 }
 
+type AlertRuleShort struct {
```

Comment:
What about something like this?

```suggestion
type SchedulableAlertRule struct {
```